server/app/routes/forms/index.js

```diff
@@ -9,6 +9,7 @@ const Questions = mongoose.model('Questions');
 const GetTheHandbook = mongoose.model('GetTheHandbook');
 const ContactRequest = mongoose.model('ContactRequest');
 const Nominate = mongoose.model('Nominate');
+// const BecomeAMentor = mongoose.model('BecomeAMentor');
```

Comment:
@ldthorne I'm getting an error here when I uncomment the line saying the schema is not being registered. Can you take a look? I don't know why it is not being registered. I set it up the same way as all the other ones I've written. Have I forgotten a require statement or something like that somewhere?